src/main/java/org/jboss/arquillian/testcontainers/TestcontainerDescription.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The Arquillian Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.jboss.arquillian.testcontainers;
+
+import java.lang.reflect.AnnotatedElement;
+
+import org.jboss.arquillian.testcontainers.api.Testcontainer;
+import org.testcontainers.containers.GenericContainer;
+
+/**
+ * @author <a href="mailto:jperkins@redhat.com">James R. Perkins</a>
```

Comment:
An explanation of this class' raison d'etre would be nice (other classes as well) . :)